packages/amplication-server/src/core/prismaSchemaUtils/schema-utils.spec.ts

```diff
@@ -44,8 +44,17 @@ describe("schema-utils", () => {
   describe("formatFieldName", () => {
     it("should format the field name in camelCase", () => {
       (isReservedName as jest.Mock).mockReturnValueOnce(false);
-      const result = formatFieldName("test_field");
-      expect(result).toEqual("testField");
+      expect(formatFieldName("Test_5_Rank_X_out_of_Y")).toEqual(
+        "test5RankXOutOfY"
+      );
+      expect(formatFieldName("ABC_Review_Form_num7_Comments")).toEqual(
+        "abcReviewFormNum7Comments"
+      );
+      expect(formatFieldName("Another_test_string")).toEqual(
+        "anotherTestString"
+      );
+      expect(formatFieldName("test")).toEqual("test");
+      expect(formatFieldName("TEST_CASE")).toEqual("testCase");
```

Comment:
It is missing the edge case for empty string. Please add one for it.

Reply:
I don't think it would check anything: formatFieldName("")).toEqual("") ?

Reply:
it would check that it won't throw an exception

Reply:
it cannot be an empty string